.gitignore

```diff
@@ -16,6 +16,7 @@ test
 
 # sh file for bash completion
 *.sh
+!install-manifests/cli/install.sh
```

Comment:
What is this for?

Reply:
*.sh at line 18, means to exclude all the .sh files
As we now have an sh file, the (!) operator is used for adding some exceptions